config/development.json

```diff
@@ -193,6 +193,7 @@
 		"stats/paid-wpcom-v2": true,
 		"stats/paid-wpcom-v3": true,
 		"stats/real-time-tab": true,
+		"stats/navigation-improvement": false,
```

Comment:
Question: Will we be enabling this by default on the development environments, like the real-time tab, sometime later?

Reply:
Yes that's right. Excellent question - we tend to set them all as false to begin with so they are only activated by intentional feature flag usage in the query string. Later down the line we flip some switches to true 👍